pontoon/api/tests/test_urls.py

```diff
@@ -47,22 +47,22 @@ def test_graphql_prod_get(settings, projects_query, client):
 def test_graphql_prod_post(settings, projects_query, client):
     settings.DEV = False
 
-    response = client.post("/graphql", projects_query, HTTP_ACCEPT="appication/json")
+    response = client.post("/graphql/", projects_query, HTTP_ACCEPT="appication/json")
```

Comment:
```suggestion
    response = client.post("/graphql/", projects_query, HTTP_ACCEPT="application/json")
```

I'm assuming the typo is not wanted here?